### ISSUE TITLE
Would like to install gdm-settings on OpenSuSE 15.5 but there's a dependency error.

### ISSUE THREAD

**Issue by linxpatrick** [gdm-settings/gdm-settings#168]
### Description

Using OpenSuSE 15.5. Trying to install gdm-settings. There is a missing dependency for pygobjecgt but, when I install it using pip, pip says the requirement is already satisfied. I would also like to install gdm-settings on OpenSuSE 15.4 but I don't currently have a development machine for testing the installation there. Using 15.5 because I will be upgrading all of my machines to 15.5 soon.

The problem started with glib2 missing but I figured that out and also installed the python development pattern and cmake for OpenSuse.

Here's the installation log:

Build started at 2023-06-29T14:11:57.693237
Main binary: /usr/bin/python3
Build Options: 
Python system: Linux
The Meson build system
Version: 0.61.4
Source dir: /home/INT/pheadley/Downloads/gdm-settings-main
Build dir: /home/INT/pheadley/Downloads/gdm-settings-main/build
Build type: native build
Project name: gdm-settings
Project version: 4.alpha
Build machine cpu family: x86_64
Build machine cpu: x86_64
Host machine cpu family: x86_64
Host machine cpu: x86_64
Target machine cpu family: x86_64
Target machine cpu: x86_64
Pkg-config binary for 1 is not cached.
Pkg-config binary missing from cross or native file, or env var undefined.
Trying a default Pkg-config fallback at pkg-config
Found pkg-config: /usr/bin/pkg-config (0.29.2)
Determining dependency 'glib-2.0' with pkg-config executable '/usr/bin/pkg-config'
env[PKG_CONFIG_PATH]: 
Called `/usr/bin/pkg-config --modversion glib-2.0` -> 0
2.70.5
env[PKG_CONFIG_PATH]: 
Called `/usr/bin/pkg-config --cflags glib-2.0` -> 0
-I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include
env[PKG_CONFIG_ALLOW_SYSTEM_LIBS]: 1
env[PKG_CONFIG_PATH]: 
Called `/usr/bin/pkg-config --libs glib-2.0` -> 0
-L/usr/lib64 -lglib-2.0
env[PKG_CONFIG_PATH]: 
Called `/usr/bin/pkg-config --libs glib-2.0` -> 0
-lglib-2.0
Run-time dependency glib-2.0 found: YES 2.70.5
Pkg-config binary for 1 is cached.
Determining dependency 'pygobject-3.0' with pkg-config executable '/usr/bin/pkg-config'
env[PKG_CONFIG_PATH]: 
Called `/usr/bin/pkg-config --modversion pygobject-3.0` -> 1

CMake binary for 1 is not cached
CMake binary missing from cross or native file, or env var undefined.
Trying a default CMake fallback at cmake
Found CMake: /usr/bin/cmake (3.20.4)
Extracting basic cmake information
Try CMake generator: auto
Calling CMake (['/usr/bin/cmake']) in /home/INT/pheadley/Downloads/gdm-settings-main/build/meson-private/cmake_pygobject-3.0 with:
  - "--trace-expand"
  - "--trace-format=json-v1"
  - "--no-warn-unused-cli"
  - "--trace-redirect=cmake_trace.txt"
  - "-DCMAKE_TOOLCHAIN_FILE=/home/INT/pheadley/Downloads/gdm-settings-main/build/meson-private/cmake_pygobject-3.0/CMakeMesonToolchainFile.cmake"
  - "."
  -- Module search paths:    ['/', '/opt', '/usr', '/usr/local']
  -- CMake root:             /usr/share/cmake
  -- CMake architectures:    []
  -- CMake lib search paths: ['lib', 'lib32', 'lib64', 'libx32', 'share']
Preliminary CMake check failed. Aborting.
Run-time dependency pygobject-3.0 found: NO (tried pkgconfig and cmake)

**Comment by realmazharhussain** [comment]
Come on, man. I even provided a link to the bug report you were supposed to fill and you still chose to use the "Feature Request" issue template?

Anyway, It seems like pygobject is not installed. From https://pygobject.readthedocs.io/en/latest/getting_started.html#opensuse-logo-opensuse, I get that you need to run `sudo zypper install python3-gobject python3-gobject-Gdk typelib-1_0-Gtk-4_0 libgtk-4-1` in order to install it.

**Comment by linxpatrick** [comment]
My apologies on creating the new request. I was confused by the closing 
of the original post.

I ran the zypper command below and everything is already installed. Any 
other suggestions to try?

*/Patrick Headley/*
Linx Consulting, Inc.
(303) 916-5522
***@***.***
www.linxco-inc.com

**Comment by Sudococommunity** [comment]
i have dependency issue i dont know how to install that which polkit 
can some help me please

**Comment by realmazharhussain** [comment]
@linxpatrick

> My apologies on creating the new request. I was confused by the closing
of the original post.

I didn't just close the issue without any instructions. I closed it with [a comment](https://github.com/gdm-settings/gdm-settings/issues/167#issuecomment-1612825370) in which I provided the link and specifically mentioned that you were supposed to fill a "bug report".

When you go to open a new issue, it gives you three options; "Bug Report", "Feature Request", and "Support Questions". I did ask you to open a new issue. But it was supposed to be a "Bug Report" not a "Feature Request".

Anyway, there is no need to open yet another issue. The discussion will continue here.

> I ran the zypper command below and everything is already installed. Any
other suggestions to try?

It seems like there is something wrong with your system. What error does installing the flatpak give?
